client/login/wp-login/style.scss

```diff
@@ -331,3 +342,22 @@ $image-height: 47px;
 		}
 	}
 }
+
+.layout.is-gutenboarding-login {
+	background-color: #fff;
+
+	.login__form-header {
+		font-family: Recoleta;
```

Comment:
Should we try to match the font-sizes to the Figma here too, or does that get a bit more complex with trying to change the size of containers, etc? The font sizes I used in the signup form are:

```
@mixin onboarding-heading-text {
	@include onboarding-font-recoleta;
	font-size: 42px;
	line-height: 57px;
}

@mixin onboarding-heading-text-mobile {
	@include onboarding-font-recoleta;
	font-size: 32px;
	line-height: 40px;
}
```

Reply:
I switched to this. It makes the heading text wrap, but it's going to wrap in non-english languages anyway.